packages/nextjs/components/scaffold-eth/Contract/ContractUI.tsx

```diff
@@ -54,66 +55,76 @@ const ContractUI = ({ contractName }: TContractUIProps) => {
     [contract, displayedContractFunctions],
   );
 
+  if (deployedContractLoading) {
+    return (
+      <div className="mt-14">
+        <Spinner width="50px" height="50px" />
+      </div>
+    );
+  }
+
   if (!contractAddress) {
-    return <p className="text-2xl">No Contract found!</p>;
+    return <p className="text-3xl mt-14">No Contract found!</p>;
   }
 
   return (
-    <div className="col-span-5 grid grid-cols-1 lg:grid-cols-3 gap-10">
-      <div className="col-span-1 flex flex-col">
-        <div className="bg-base-100 border-base-300 border shadow-md shadow-secondary rounded-3xl px-8 mb-6 space-y-1 py-4">
-          <div className="flex">
-            <div className="flex flex-col gap-1">
-              <Address address={contractAddress} />
-              <div className="flex gap-1 items-center">
-                <span className="font-bold text-sm">Balance:</span>
-                <Balance address={contractAddress} className="px-0 h-1.5 min-h-[0.375rem]" />
+    <div className="grid grid-cols-1 lg:grid-cols-6 py-12 px-10 lg:gap-12 w-full max-w-7xl my-0">
```

Comment:
The git diffing algorithm is showing lots of changes in this return JSX 

Actually what I did was instead of wrapping <ContractUI /> in `page/debug.tsx` with `<div className="grid grid-cols-1...`  I wrapped  the JSX inside ContractUI with that `<div className="grid grid-cols-1...` itself

Since it made more sense because `<div className="grid grid-cols-1...` intend was layout different blocks inside ContractUI and also helped us style "No Contract Found"

Reply:
A trick I use when reviewing PRs to avoid this kind of thing, "hide whitespace".

![image](https://user-images.githubusercontent.com/2486142/220159655-de6dbfca-694c-4439-bf12-ef8fb9846d0e.png)